examples/pg_proxy.py

```diff
@@ -25,68 +31,53 @@ def t(row):
         return iter(map(t, self._rows))
 
 
-class PGAdapter(Adapter):
-    pg8000.dbapi.paramstyle = "numeric"
-
-    def __init__(self, **kwargs):
-        self.db = pg8000.dbapi.connect(**kwargs)
-        self.db.autocommit = True
-
-    def _cursor(self):
-        return self.db.cursor()
-
-    def parameters(self) -> Dict[str, str]:
-        return {"server_version": "BV.pg8000.1", "client_encoding": "UTF8"}
+class PGAdapterHandle(AdapterHandle):
+    def __init__(self, cursor):
+        super().__init__(cursor)
```

Comment:
```suggestion
```

Don't need to redefine the init if it's identical

Reply:
Ack thanks! Still not very good at Python 😉